### ISSUE TITLE
Corrupted image if req.file('file').upload is chained

### ISSUE THREAD

**Issue by toovy** [balderdashy/skipper-s3#43]
I'm trying to accomplish a upload to S3 with thumbnail resizing. As this is not yet implemented I'm struggling with it. So the first step was to try to upload the same image twice (without resizing). The file gets uploaded twice, but one version is slightly smaller in file size and corrupted.

    confA =
      adapter: require('skipper-s3')
      key: process.env.S3_KEY
      secret: process.env.S3_SECRET
      bucket: process.env.USI
      region: process.env.S3_REGION
      dirname: 'asd'
      headers: { 'x-amz-acl': 'public-read' }
    confB =
      adapter: require('skipper-s3')
      key: process.env.S3_KEY
      secret: process.env.S3_SECRET
      bucket: process.env.USI
      region: process.env.S3_REGION
      dirname: 'asd'
      headers: { 'x-amz-acl': 'public-read' }
    req.file('file')
    .upload confA, (err, files) ->
      console.log "file 1 ok"
    .upload confB, (err, files) ->
      console.log "file 2 ok"

With a simple custom receiver like there http://stackoverflow.com/questions/24069203/skipper-in-sailsjs-beta-image-resize-before-upload chaining works. I could even accomplish the thumbnail resizing. So I guess there is a bug in the skipper-s3 adapter.

**Comment by sailsbot** [comment]
Thanks for posting, @toovy.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
